.github/workflows/ci.yaml

```diff
@@ -323,6 +347,8 @@ jobs:
   test_rust:
     name: 'Test :: Rust'
     runs-on: ${{ matrix.info.image }}
+    outputs:
+        FRB_MSRV_RUST_VERSION: ${{ env.FRB_MSRV_RUST_VERSION }}
```

Comment:
(this seems to be redundant and needs to be removed)

Reply:
Oh yes! This is a left-over of trying to define the variable on top of the CI file - removed it now.